monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+
+Dans le cas des algorithmes MLSAG/CLSAG utilisés par Monero, le délai de traitement de la transaction est proportionnel à sa taille: les deux augmentent linéairement en fonction du nombre de leurres utilisés. L’utilisation de dix leurres permet d'atteindre un niveau d'anonymat acceptable, contrebalancé par le besoin d’une chaine de blocs plus petite permettant un traitement plus expéditif des transactions. Un plus grand nombre de leurres offrirait une anonymisation supérieure, mais augmenterait également la taille de la chaine de blocs de Monero – déjà plusieurs fois plus grande que celle de Bitcoin - et ralentirait les portefeuilles électroniques lors des validations de transaction.
+
+### _Triptych_
+
+Triptych a été conçu pour adresser ces défis: il s’agit d’un nouvel algorithme destiné aux signatures de cercles développé par Sarang Noether et Brandon Goodell des Laboratoires de Recherche de Monero (MRL) avec la participation de Arthur Blue (RandomRun). Avec Triptych, la taille de la signature de cercle croît de manière logarithmique plutôt que linéaire. La croissance logarithmique est très lente, et présente un choix judicieux qui permet d’éviter de faire confiance à des tierces parties (et par là, faire échec à l’anonymat comme pour Zcash). Le délai de vérification des opérations croît toujours linéairement avec le nombre de leurres avec Triptych, mais les vérifications peuvent-être regroupées et triées par l'entremise d’algorithmes spécialisés, accélérant le traitement.
+
+Triptych parvient à faire fi de plusieurs difficultés posées par l’usage des signatures de cercles avec Monero. Il permet notamment d’établir un lien: la technique des signatures de cercles de Monero doit bloquer la réutilisation de clés publiques légitimes afin de prévenir le double usage d’un TXO. Triptych y parvient en utilisant des étiquettes de liaison ("linking tags"), similaires aux images cryptographiques actuelles. Triptych est aussi compatible avec les transactions confidentielles, qui permettent d’obfusquer les volumes de Monero échangés. Des données spéciales (“clés d’engagement - "commitment keys"), permettent de valider toutes les opérations de mouvement de Monero sans dévoiler leurs montants. Ces clés d’engagement, avec les clés publiques et les étiquettes de liaison, sont les trois éléments qui donnent naissance au nom de Triptych, qui signifie "formé de trois parties".
+
+### _Tests et Alternatives_
+
+Triptych permet une augmentation quantifiable de la rapidité d'utilisation de Monero. Un de ses créateurs, Sarang Noether, a signalé que le code d’essai donnait une amélioration d’exécution significative. En bref, l’utilisation de 63 leurres sous Tryptych est 10% plus rapide que l’utilisation de 10 leurres avec CLSAG. Ces performances présentent une amélioration fondamentale pour Monero, tout en étant pratique à l'utilisation.
```

Comment:
En bref >>> "Grossièrement" or "De manière approximative"
Reason:
The original term 'Roughly' is more used in the context of 'Approximately' and not 'To summarize'

Reply:
Integrated grossièrement.